tests/OperationStore.spec.ts

```diff
@@ -125,6 +125,31 @@ describe('OperationStore', async () => {
     expect(returnedOperationHash).toEqual(updateOperation.getOperationHash());
   });
 
+  it('should ignore duplicate updates', async () => {
```

Comment:
Sanity: aren't all operation store unit tests pointless since they are testing just the mock store?

Reply:
Yes and no. I use them with minor edits to test MongoDbOperationStore before I checkin any changes. If you have better ideas for testing MongoDbOperationStore, please suggest.

Reply:
I get how you would be testing MongoDbOperationStore with this class, but this would be unknown/hidden to people who does not know that this code exists.

We can add a funciton in the `OperationStore.spec.ts`:

```javascript
function skipTestIfMongoDbNotConfigured (): boolean {
  const mongoDbFound = ...;
  if (!mongoDbFound) {
    pending('MongoDB not configured');
  }
}
```

then call this method in each of the `OperationStore` tests.

Essentially we are introducing a new class of tests that are not unit tests.

Reply:
Talked to Arvind offline, this will be addressed in PR #170.